docs/Customizing_site.rst

```diff
@@ -4167,7 +4172,7 @@ Assume that you want to remove the **Help** portlet from the top
 navigation bar, do as follows:
 
 1. Remove the following block from
-   ``custom-extension.war!/WEB-INF/conf/portal/portal/sharedlayout.xml``.
+   ``custom-extension.war!/WEB-INF/conf/portal/portal/sharedlayout-<SITENAME>.xml``.
```

Comment:
The xml block does not exist in the shared layout of the dw site, so it can be confusing for the reader.

Reply:
done